src/test/java/com/programmers/springbootboardjpa/domain/post/controller/PostControllerTest.java

```diff
@@ -0,0 +1,197 @@
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.programmers.springbootboardjpa.domain.post.dto.PostCreateRequestDto;
+import com.programmers.springbootboardjpa.domain.post.dto.PostResponseDto;
+import com.programmers.springbootboardjpa.domain.post.dto.PostUpdateRequestDto;
+import com.programmers.springbootboardjpa.domain.post.service.PostService;
+import com.programmers.springbootboardjpa.domain.user.dto.UserRequestDto;
+import com.programmers.springbootboardjpa.domain.user.dto.UserResponseDto;
+import com.programmers.springbootboardjpa.domain.user.service.UserService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.restdocs.AutoConfigureRestDocs;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.restdocs.payload.JsonFieldType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.transaction.annotation.Transactional;
+
+import static org.springframework.restdocs.mockmvc.MockMvcRestDocumentation.document;
+import static org.springframework.restdocs.mockmvc.RestDocumentationRequestBuilders.*;
+import static org.springframework.restdocs.payload.PayloadDocumentation.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@AutoConfigureMockMvc
+@AutoConfigureRestDocs
+@Transactional
+@SpringBootTest
+class PostControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Autowired
+    private PostService postService;
+
+    @Autowired
+    private UserService userService;
+
+    private PostCreateRequestDto postCreateRequestDto;
+
+    @BeforeEach
+    void setUp() {
+        UserRequestDto userRequestDto = UserRequestDto.builder()
+                .name("김이름")
+                .age(27)
+                .hobby("산책")
+                .build();
+
+        UserResponseDto userResponseDto = userService.create(userRequestDto);
+        Long userId = userResponseDto.id();
+
+        postCreateRequestDto = PostCreateRequestDto.builder()
+                .title("등록용 제목")
+                .content("등록용 내용")
+                .userId(userId)
+                .build();
+    }
+
+    @DisplayName("게시글을 저장한다")
+    @Test
+    void create() throws Exception {
+        //given
+        //when
+        //then
+        mockMvc.perform(post("/api/v1/posts")
+                        .content(objectMapper.writeValueAsString(postCreateRequestDto))
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andDo(document("post-create",
+                        requestFields(
```

Comment:
```suggestion
                .andDo(document("post-create",
                                preprocessRequest(prettyPrint()),
                                preprocessResponse(prettyPrint()),
                        requestFields(....
```
취향일수 있지만 요청,응답에 `prettyPrint()`를 넣어주면 RestDocs에서 Josn이 줄바꿈이 되어 조금 더 가독성이 높아져요!

Reply:
추가했습니다! 감사합니다 😊